README.md

```diff
@@ -70,77 +99,9 @@ After the client is created, you can start sending events to your Datadog Event
 
 
-## Usage via the Statsd class:
-
-In most cases, the static DogStatsd class is probably better to use. However, the Statsd is useful when you want to queue up a number of metrics/events to be sent in one UDP message (via the `Add` method).
-
-``` C#
-// The code is located under the StatsdClient namespace
-using StatsdClient;
-
-// ...
-
-// NB: StatsdUDP is IDisposable and if not disposed, will leak resources
-StatsdUDP udp = new StatsdUDP(HOSTNAME, PORT);
-using (udp)
-{
-  Statsd s = new Statsd(udp);
-
-  // Incrementing a counter by 1
-  s.Send<Statsd.Counting,int>("stat-name", 1);
-
-  // Recording a gauge
-  s.Send<Statsd.Gauge,double>("stat-name", 5.5);
-
-  // Sampling a histogram
-  s.Send<Statsd.Histogram,int>("stat-name", 1);
-
-  // Measure a distribution of values
-  s.Send<Statsd.Distribution,int>("stat-name", 1);
-
-  // Send elements to a set
-  s.Send<Statsd.Set,int>("stat-name", 1);
-  s.Send<Statsd.Set,string>("stat-name", "stat-value");
-
-  // Send a timer
-  s.Send<Statsd.Timing,double>("stat-name", 3.1337);
-
-  // Time a method
-  s.Send(() => MethodToTime(), "stat-name");
-
-  // See note below on how exceptions in timed methods are handled
-
-  // All types have optional sample rates and tags:
-  s.Send<Statsd.Counting,int>("stat-name", 1, sampleRate: 1/10, tags: new[] {"tag1:true", "tag2"});
-
-  // Send an event
-  s.Send("title", "content");
-
-  // You can add combinations of messages which will be sent in one go:
-  s.Add<Statsd.Counting,int>("stat-name", 1);
-  s.Add<Statsd.Timing,int>("stat-name", 5, sampleRate: 1/10);
-  s.Add("event title", "content", priority: "low");
-  s.Send(); // message will contain counter and will contain timer 10% of the time
-  // All previous commands will be flushed after any Send
-  // Any Adds will be ignored if using a Send directly
-  s.Add<Statsd.Counting,int>("stat-name", 1);
-  s.Send<Statsd.Timing,double>("stat-name", 4.4); // message will only contain Timer
-  s.Send(); // the counter will not be sent by the command
-}
-
-// By default, Statsd will split messages containing multiple metrics/events into
-// UDP messages that are 512 bytes long. To change this limit, create a new
-// instance of StatsUDP
-int maxUDPPacketSize = 4096;
-StatsUDP udpNew = new StatsdUDP(HOSTNAME, PORT, maxUDPPacketSize);
-using (udP)
-{
-  // ...
-}
-// To disable the splitting of UDP messages, set this limit to 0
-```
+## Usage via the Statsd class
 
-A note about timing: Statsd will not attempt to handle any exceptions that occur in a timed method. If an unhandled exception is thrown while timing, a timer metric containing the time elapsed before the exception occurred will be sent or added to the send queue (depending on whether Send or Add is being called).
+`Statsd` is marked as obsolete as both `DogStatsdService` and `DogStatsd` methods do not block anymore and batch several metrics automatically in one UDP or UDS message.
```

Comment:
I am wondering if we should leave the snippet (or a simplified version of it) but say that it is deprecated for the reasons named above. I'm not sure, what do you think? 🤔

Reply:
I do not have a strong opinion on this topic but ideally we would like making `Statsd` private as soon as possible (to make changes to this class without doing breaking changes). Removing this section from README.md should reduce its usage and accelerate the migration.